cabal-install/src/Distribution/Client/ProjectConfig.hs

```diff
@@ -559,18 +559,15 @@ readProjectConfig :: Verbosity
                   -> Flag FilePath
                   -> DistDirLayout
                   -> Rebuild ProjectConfigSkeleton
-readProjectConfig verbosity httpTransport ignoreProjectFlag configFileFlag distDirLayout = do
-    global <- singletonProjectConfigSkeleton <$> readGlobalConfig verbosity configFileFlag
-    local  <- readProjectLocalConfigOrDefault verbosity httpTransport distDirLayout
-    freeze <- readProjectLocalFreezeConfig    verbosity httpTransport distDirLayout
-    extra  <- readProjectLocalExtraConfig     verbosity httpTransport distDirLayout
-    if ignoreProjectFlag == Flag True then return (global <> (singletonProjectConfigSkeleton defaultProject))
-    else return (global <> local <> freeze <> extra)
-    where
-      defaultProject :: ProjectConfig
-      defaultProject = mempty {
-        projectPackages = ["./"]
-      }
+readProjectConfig verbosity _ (Flag True) configFileFlag _ = do
+      global <- singletonProjectConfigSkeleton <$> readGlobalConfig verbosity configFileFlag
```

Comment:
Does it re-indent old code for some reason?

Reply:
I didn't notice that overall the intentation has changed.

Reply:
Maybe it's the artifact of how GitHub presents it. I'm just surprised that there's any diff produced here. Oh, well...